libscpi/inc/scpi/minimal.h

```diff
@@ -59,7 +59,7 @@ extern "C" {
     scpi_result_t SCPI_StatusOperationEnableQ(scpi_t * context);
     scpi_result_t SCPI_StatusOperationEnable(scpi_t * context);
     scpi_result_t SCPI_StatusPreset(scpi_t * context);
-
+    scpi_result_t SCPI_HelpQ(scpi_t * context);
```

Comment:
I would like to keep `minimal.c/h` really minimal implementation of mandatory SCPI commands which `help` is not. Can you please provide new `.c/.h` for this. There are already similar extensions like `expression.c/h`, `error.c/h`. So, it can be simply `help.c/h`.